src/components/FormHtmlEditor.vue

```diff
@@ -63,14 +63,20 @@ export default {
       }
 
       try {
-        return Mustache.render(this.content, this.validationData);
+        return Mustache.render(this.content, Object.assign({}, this.customFunctions, this.validationData));
```

Comment:
I think this would read better using destructuring:
```suggestion
        return Mustache.render(this.content, { ...this.customFunctions, ...this.validationData});
```

Reply:
Changed